app/View/Pages/contact.ctp

```diff
@@ -93,20 +93,34 @@ $this->set('title_for_layout', $this->Pages->formatTitle(__('Contact us')));
     
     
     <div class="module">
-        <h2><?php echo __('Join us on IRC'); ?></h2>
+        <h2><?php echo __('Join us on XMPP'); ?></h2>
+        <p>
         <?php
         echo format(
             __(
-                'We also have an IRC channel on freenode, #tatoeba. If you are not '.
-                'familiar with IRC, you can read the <a href="{}">Help</a>.', true
+                'We also have an XMPP room, <a href="{room}">'.
+                'tatoeba@chat.tatoeba.org</a>, also available with the web '.
+                'client below.  If you are not familiar with XMPP, you '.
+                'can read the <a href="{help}">Help</a>.', true
             ),
-            $this->Html->url(
-                array(
-                    "controller" => "pages",
-                    "action" => "help"
+            array(
+                'room' => "xmpp:tatoeba@chat.tatoeba.org?join"
```

Comment:
There's a comma missing at the end. But don't push any fix yet. We're gonna need to discuss about what to write on this Contact page :)